audit/ask-cal/vitest.setup.ts

```diff
@@ -0,0 +1,21 @@
+import "@testing-library/jest-dom";
+import { cleanup } from "@testing-library/react";
+import { afterEach } from "vitest";
+
+// We overwrite console methods to suppress certain errors that are expected & distracting
+const originalConsoleError = console.error;
+console.error = function (...data) {
+  // JSDom uses an outdated CSS parser that throws errors on valid modern CSS
```

Comment:
Lol, oh no, this sounds like a rabbit hole!

Reply:
yeah... thankfully I didn't need to go down that rabbit hole. Ben and Nick already did! I referenced their work with the chatbot, as it uses a similar architecture for the UI.